runner/lib/tasks/local-chain.js

```diff
@@ -232,6 +234,11 @@ export const makeTasks = ({
 
         await PromiseAllOrErrors([outputParsed, launcherDone]);
       }
+
+      const genesis = JSON.parse(
+        await fs.readFile(genesisPath, { encoding: 'utf-8' }),
```

Comment:
we don't worry about nested await in this repo, I gather...

Reply:
Maybe some day I will, but it's particularly hard in this kind of runner code, and not worth the effort IMO.